eth_account/account.py

```diff
@@ -34,14 +34,14 @@
 from eth_account.datastructures import (
```

Comment:
https://circleci.com/gh/ethereum/eth-account/1209?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

This import needs to be moved down below the `_utils` imports.

Reply:
Done!